docs/html/development/issue-triage.rst

```diff
@@ -0,0 +1,305 @@
+============
+Issue Triage
+============
+
+This serves as an introduction to issue tracking in pip as well as
+how to help triage reported issues.
+
+
+Issue Tracker
+*************
+
+The `pip issue tracker <https://github.com/pypa/pip/issues>`__ is hosted on
+GitHub alongside the project.
+
+Currently, the issue tracker is used for bugs, feature requests, and general
+user support.
+
+In the pip issue tracker, we make use of labels and milestones to organize and
+track work.
+
+
+Labels
+------
+
+Issue labels are used to:
+
+#. Categorize issues
+#. Provide status information for contributors and reporters
+#. Help contributors find tasks to work on
+
+The current set of labels are divided into several categories identified by
+prefix:
+
+**C - Category**
+  which area of ``pip`` functionality a feature request or issue is related to
+
+**K - Kind**
+
+**O - Operating System**
+  for issues that are OS-specific
+
+**P - Project/Platform**
+  related to something external to ``pip``
+
+**R - Resolution**
+  no more discussion is really needed, an action has been identified and the
+  issue is waiting or closed
+
+**S - State**
+  for some automatic labels and other indicators that work is needed
+
+**type**
+  the role or flavor of an issue
+
+The specific labels falling into each category have a description that can be
+seen on the `Labels <https://github.com/pypa/pip/labels>`__ page.
+
+In addition, there are several standalone labels:
+
+**good first issue**
+  this label marks an issue as beginner-friendly and shows up in banners that
+  GitHub displays for first-time visitors to the repository
+
+**triage**
+  default label given to issues when they are created
+
+**trivial**
+  special label that when used on a PR removes the news file requirement
```

Comment:
I'm not a native speaker but this sounds weirdly phrased.
`special label for pull requests that removes the news file requirement` (maybe with a link to https://github.com/pypa/pip/blob/master/docs/html/development/contributing.rst#choosing-the-type-of-news-entry ?

Reply:
Thanks for the feedback! I was missing a few commas here, but I think your wording is better. Updated [here](https://github.com/pypa/pip/pull/6868/files#diff-0ee19f321d04652294c191edc037e9a7R68-R69).